xcm/xcm-simulator/example/src/lib.rs

```diff
@@ -60,40 +61,68 @@ decl_test_network! {
 	}
 }
 
-pub fn para_account_id(id: u32) -> relay_chain::AccountId {
-	ParaId::from(id).into_account()
+pub fn parent_account_id() -> parachain::AccountId {
+	let location = (Parent,);
+	parachain::LocationToAccountId::convert(location.into()).unwrap()
+}
+
+pub fn child_account_id(para: u32) -> relay_chain::AccountId {
```

Comment:
Is this the para account ID instead?

Reply:
yes; a child parachain (rather than a sibling parachain, as it would be when addressed from another parachain which shared the same parent).